test/functionalTest/cases/2584_service_path_header_in_notifs/service_path_header_in_notifs.test

```diff
@@ -0,0 +1,196 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This is merely a tool that I needed while trying to fix the bug.
And you're right, once the bug is fixed, no need for this test anymore.

And to anticipate :-):
  No, it is not easier to try to find a 'half valid' already existing functest than to create a new one.
  A new one is both faster and better suited for the problem
  To create  a functest takes me a few minutes, copying from older functests (as I believe we all do)